app/pods/cases/case/subcases/subcase/decision/route.js

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+
+export default class CasesCaseSubcasesSubcaseDecisionRoute extends Route {
+  async model() {
+    const subcase = this.modelFor('cases.case.subcases.subcase');
+    const pieces = await this.store.query('piece', {
+      'filter[agenda-item-treatment][subcase][:id:]': subcase.id,
+    });
+    return pieces.toArray();
```

Comment:
Sortering in geval van meerdere beslissingen, dat mist hier nog.
Hoe sorteren we dat in de agendaitem detail route en kunnen we die sortering hier ook gebruiken ?

Reply:
Voor beslissingsdocumenten bestaat op heden geen custom sorteer-algoritme. Als tussenoplossing hier gesorteerd op creatiedatum van het document (descending). Zie https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/776/commits/c8ed38751c5e3b51e6eb9e62245c4e57d0eccabe